pandas/_libs/tslibs/src/datetime/np_datetime_strings.c

```diff
@@ -104,19 +119,28 @@ int parse_iso_8601_datetime(const char *str, int len, int want_exc,
     while (sublen > 0 && isspace(*substr)) {
         ++substr;
         --sublen;
+        if (!format_startswith(' ', format_len, *format, exact)) goto parse_error;
```

Comment:
Minor nit but can you run `clang-format` against this file? I think it would format this differently